ovos_tts_plugin_server/__init__.py

```diff
@@ -41,6 +41,11 @@ def verify_ssl(self) -> bool:
         """Whether or not to verify SSL certificates when connecting to the server. Defaults to True."""
         return self.config.get("verify_ssl", True)
 
+    @property
+    def tts_timeout(self) -> int:
+        """Timeout for the TTS server. Defaults to 30 seconds."""
+        return self.config.get("tts_timeout", 30)
```

Comment:
This is really long when the server times out.  I have noticed recently because of the `smart'gic` servers being down lately.  Is there a reason it is so long?

Reply:
Yes - I don't want to have the default setting ruin the user experience if they're using TTS with a slow RTF. @emphasize pointed out that this is not uncommon. Ideally we'd set this algorithmically, based on length of the string and expected RTF, but that's a bigger effort and I wanted to make this configurable out of the gate.

If you're seeing that it's particularly slow if a server is down we can adjust it though. That's probably a worse user experience

Reply:
i'd argue a long RTF is not a default use case, since a user needs to point to a url for this to be the case, its fair to reduce default value and require user to change it too together with the url

Reply:
Fair point, what's a good default then? 5s? 10s?

Reply:
I think 5s is good.